src/Game/KeyboardHandler.java

```diff
@@ -139,6 +140,15 @@ else if (e.getKeyCode() == KeyEvent.VK_9 || e.getKeyCode() == KeyEvent.VK_NUMPAD
       e.consume();
     }
 
+    // Close friend chat with escape
+    if (Client.state == Client.STATE_GAME
+        && Client.show_friends == 2
+        && e.getKeyCode() == KeyEvent.VK_ESCAPE
+        && !Replay.isPlaying) {
```

Comment:
it would be appropriate for Escape to still close the private message box during replay playback.

Reply:
Updated